docs/source/models/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -1044,6 +1044,25 @@ These covariate values are calculated for each simulant and are then plugged int
 Where sex = 1 for men and 2 for women 
 and SBP and LDL level refer to the raw values from GBD 
 
+**Adjusting for State Level Variation:** 
+
+Each state has an age/sex/medication type specific "relative risk" value for being 
+medicated. These values are stored in the 'state_medication_FAKE_data.csv'. This 
```

Comment:
Can you confirm what to do w/ missing data? Specifically, age < 25 (I know you don't actually care for this siom) and > 80? Presumably you want to backfill < 25 and forwardfill > 80.